Formula/ice.rb

```diff
@@ -19,6 +19,7 @@ class Ice < Formula
 
   depends_on "lmdb"
   depends_on "mcpp"
+  depends_on macos: :catalina
```

Comment:
Did you run `brew audit` on this? It might want you to put this line before `mcpp`.

Reply:
I did, it didn't complain.

Reply:
hmmm.. I ran it again and it failed :/. Will fix